includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -1129,6 +1023,58 @@ public function payment_fields() {
+
+	/**
+	 * Whether to hide the store-level save-payment-method checkbox.
+	 *
+	 * Base UPE hides it when Link is enabled — Link handles save consent via the Payment
+	 * Element. Subclasses may override.
+	 *
+	 * @return bool
+	 */
+	protected function should_hide_save_payment_method_checkbox(): bool {
+		return WC_Stripe_UPE_Payment_Method_Link::is_link_enabled( $this );
+	}
```

PHPMD (2.15.0)

[error] The method should_hide_save_payment_method_checkbox is not named in camelCase. (undefined)

(CamelCaseMethodName)